bundles/org.openhab.binding.orbitbhyve/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="orbitbhyve"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Mode Channel Type -->
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Irrigation Mode</label>
+		<description>Channel representing mode of Orbit B-Hyve Device (auto/manual)</description>
+		<state readOnly="true"></state>
```

Comment:
You could specify the two options within the state tag.

Reply:
added both suggestions